visual-tests/tests/copy-cell-content.spec.js

```diff
@@ -0,0 +1,46 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+import { helpers } from '../imports/helpers';
+
+const stylesToAdd = [
+  helpers.cssFiles.cookieInfo,
+  helpers.cssFiles.dynamicDataFreeze
+];
+
+test(helpers.testTitle(path.basename(__filename)), async({ page }, workerInfo) => {
+  helpers.init(workerInfo);
+  await page.goto(helpers.testURL);
+  await expect(page).toHaveTitle(helpers.expectedPageTitle);
+  stylesToAdd.forEach(item => page.addStyleTag({ path: helpers.cssPath(item) }));
+
+  const table = page.locator(helpers.selectors.mainTable);
+
+  await table.waitFor();
+  helpers.mainTableFirstColumn = table.locator(helpers.selectors.mainTableFirstColumn);
+  helpers.tbody = table.locator(helpers.selectors.mainTableBody);
+  helpers.thead = table.locator(helpers.selectors.mainTableHead);
+
+  let cell = helpers.tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));
+
+  await cell.click();
+  await page.screenshot({ path: helpers.screenshotPath() });
+  await page.keyboard.press(`${helpers.modifier}+c`);
+  await cell.press('Delete');
+
+  cell = helpers.tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));
+  await cell.dblclick();
+  await cell.type('-test');
+
+  cell = helpers.tbody.locator(helpers.findCell({ row: 4, cell: 3, cellType: 'td' }));
+  await cell.click();
+  await page.keyboard.press('Delete');
+
+  cell = helpers.tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));
+  await cell.click();
```

Comment:
```suggestion
  await table.waitFor();
  
  const tbody = table.locator(helpers.selectors.mainTableBody);

  let cell = tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));

  await cell.click();
  await page.screenshot({ path: helpers.screenshotPath() });
  await page.keyboard.press(`${helpers.modifier}+c`);
  await cell.press('Delete');

  cell = tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));

  await cell.dblclick();
  await cell.type('-test');

  cell = tbody.locator(helpers.findCell({ row: 4, cell: 3, cellType: 'td' }));

  await cell.click();
  await page.keyboard.press('Delete');

  cell = tbody.locator(helpers.findCell({ row: 3, cell: 3, cellType: 'td' }));

  await cell.click();
```
I think that it's cleaner to just assign the elements to the local variables and use them within the test instead of assigning them to helpers utils.

Reply:
The same thing appears in few files - it was done intentionally as a compromise to make tester's life easier, now should be modified as we discussed.